src/main/groovy/edu/oregonstate/mist/locations/LocationUtil.groovy

```diff
@@ -0,0 +1,99 @@
+package edu.oregonstate.mist.locations
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import java.security.MessageDigest
+
+class LocationUtil {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LocationUtil.class)
+
+    public static final String VALID_LAT_LONG = "-?\\d+(\\.\\d+)?"
```

Comment:
if i read this right, `-1`, `-2.3`, `45`, and `67.89` are all syntactically valid values for latitude or longitude.

semantically invalid values such as `-789` and `345.6` will also satisfy this regex.

you should confirm that units are decimal, not degrees, minutes, and seconds.

consider using `(?x)` to enable explanatory comments for maintainability.

note that although the regular expression string is defined as a constant, every time it is used it will be recompiled or reevaluated. compiling the expression would mitigate this. i see that the expression is only used in tests, so there should be no runtime cost. however, since the string is `public`, future users of `LocationUtil` may incur unexpected costs.